README.md

```diff
@@ -22,14 +22,53 @@ This application is a denormalised warehouse for multiple LIMS.
 
 Before you can use the system in any capacity, you must first prepare the database.
 This should be handled by `bin/setup` above, but if not:
+```
```

Comment:
Markdown doesn't need fences when indented with 4 spaces.

Reply:
I didn't know that, ok